assets/src/components/v2/train_crowding.tsx

```diff
@@ -0,0 +1,163 @@
+  const arrowSlot = [1, 9, 17].includes(platform_position) ?
+        platform_position + 1
+        : platform_position == 25 ? platform_position - 1 : platform_position
+  
+  // The slot arrangement exceeds the edges of the screen, so to properly set the slot positions
+  // we need to make a few adjustments. 
+  const screenWidth = 3240
+  const slotOverhang = 35
+  const slotsWidth = screenWidth + slotOverhang*2
+  const extraArrowPadding = 24
+  const arrowLeftPadding = (arrowSlot-1) * (slotsWidth/25) - slotOverhang - extraArrowPadding
+  const arrowDirection = [1, 9, 17].includes(platform_position) ? "up-left"
+        : platform_position == 25 ? "up-right" : "up"
+
+  const textPane = Math.floor(arrowSlot/25 * 3)
+  const textPadding = textPane * 3240/3
+
+  return <div className="crowding-widget">
+    <div className="crowding-widget__header">
+      <div className="crowding-widget__header__top-row">
+        <Logo width="128" height="128" color="#E5E4E1" className="t-logo" />
+        {now && <NormalHeaderTime time={now} />}
+      </div>
+      <div className="crowding-widget__header__destination-sentence">
+        Next train to<span className="destination">{destination}</span>
+      </div>
+    </div>
+    <div className="crowding-widget__body">
+      <div className="crowding-widget__train-row">
+        {trainSequence}
+      </div>
+      <div style={{paddingLeft:arrowLeftPadding}}>
+        { arrowDirection == "up" ?
+            <ArrowUp className={classWithModifier("crowding-widget__you-are-here-arrow", arrowDirection)} />
+            : <ArrowUpLeft className={classWithModifier("crowding-widget__you-are-here-arrow", arrowDirection)} /> }
+      </div>
+      <div style={{marginLeft:textPadding}} className={
+            classWithModifier(
+                "crowding-widget__you-are-here-text",
+                [8, 16, 24].includes(arrowSlot) ? "right-align" : "left-align"
+        )}>You are here</div>
+    </div>
+    <div className="crowding-widget__footer">
+      <div className="crowding-widget__footer__segment">
+        Current crowding on board
+      </div>
+      <div className="crowding-widget__footer__segment">
+        <div className="crowding-widget__footer__key-row">
+          <KeyNotCrowded width="137" height="100" className="key-icon" /> Not crowded
+        </div>
+        <div className="crowding-widget__footer__key-row">
+          <KeySomeCrowding width="137" height="100" className="key-icon" /> Some crowding
+        </div>
+      </div>
+      <div className="crowding-widget__footer__segment">
+        <div className="crowding-widget__footer__key-row">
+          <KeyCrowded width="137" height="100" className="key-icon" /> Crowded
+        </div>
+        <div className="crowding-widget__footer__key-row">
+          {
+            crowding.includes("closed") ?
+                <><KeyClosed width="137" height="100" className="key-icon" /> Closed</>
```

Comment:
Based on the "Specs + Components" figma page, it looks like this should be "Car closed"

Reply:
Ooo sneaky good catch